### ISSUE TITLE
junit testcase created for empty yaml docs

### ISSUE THREAD

**Issue by davidholsgrove** [yannh/kubeconform#45]
Testing out the junit xml output format, I see we have empty testcases added to the report where kubeconform encounters a yaml document with no content, including where its just a comment - for example;

```yaml
# https://raw.githubusercontent.com/vmware-tanzu/helm-charts/velero-2.14.7/charts/velero/crds/restores.yaml
---
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  labels:
    app.kubernetes.io/name: velero
  annotations:
    controller-gen.kubebuilder.io/version: v0.3.0
  name: restores.velero.io
```

```bash
kubeconform -strict -output junit velero/crds/restores.yaml
```

produces a testsuite with 2 testcases;

```xml
<testsuites name="kubeconform" time="0.3551641" tests="1" failures="0" disabled="0" errors="1">
  <testsuite name="velero/crds/restores.yaml" id="1" tests="2" failures="0" errors="1" disabled="0" skipped="0">
    <properties></properties>
    <testcase name="" classname="@"></testcase>
    <testcase name="restores.velero.io" classname="CustomResourceDefinition@apiextensions.k8s.io/v1beta1">
      <error message="could not find schema for CustomResourceDefinition" type=""></error>
    </testcase>
  </testsuite>
</testsuites>
```

**Comment by yannh** [comment]
Hi @davidholsgrove , It's tricky but it actually seem to work as expected... at least if my understanding of yaml is correct :) the --- is a separator, your file contains two yaml documents: one empty one, that contains only a comment - and another, normal, one. 

I guess the question is what Junit output we expect for empty documents :thinking:

**Comment by yannh** [comment]
I have rewritten the Junit output tests, and I believe I have fixed this. 

https://github.com/yannh/kubeconform/commit/2eefa7fc224aaff3fe3a687c9b33c4ed1d5432ac

I will make a new release very soon.

**Comment by yannh** [comment]
Released with v0.4.10.
